wallet/createtx.go

```diff
@@ -321,9 +321,22 @@ func (w *Wallet) txToOutputsInternal(op errors.Op, outputs []*wire.TxOut, accoun
 			account: account,
 			wallet:  w,
 		}
+
 		var err error
-		atx, err = txauthor.NewUnsignedTransaction(outputs, txFee,
-			inputSource.SelectInputs, changeSource)
+
+		if recipientPaysFee {
+			if len(outputs) != 1 {
+				return errors.E(op, "Expected exactly one output for "+
```

Comment:
This is an `errors.Invalid` kind of error.  Also keep the error messages lower cased for consistency.